lib/filesystem.js

```diff
@@ -73,9 +73,16 @@ FileSystem.prototype.getItem = function(filepath) {
   var parts = getPathParts(filepath);
   var currentParts = getPathParts(process.cwd());
   var item = this._root;
+  var itemPath = '/';
+  var linkPath;
   var name;
   for (var i = 0, ii = parts.length; i < ii; ++i) {
     name = parts[i];
+    if (item instanceof SymbolicLink) {
+      // Symbolic link being traversed as a directory
+      linkPath = path.resolve(path.dirname(itemPath), item.getPath());
```

Comment:
there might be an issue here with `<link> -> <link> -> <file>`-type paths or something